Project.toml

```diff
@@ -7,9 +7,13 @@ version = "1.1.3"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 Countries = "5bf49175-83fb-4b6d-82b5-d6ded85154ae"
 DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 GeoIO = "f5a160d5-e41d-4189-8b61-d57781c419e3"
+GeoJSON = "61d90e0f-e114-555e-ac52-39dfb47a3ef9"
 GeoTables = "e502b557-6362-48c1-8219-d30d308dcdb0"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 INMET = "1fb021e9-9ab7-46c1-8a23-23f3c2783cf7"
+JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
```

Comment:
there is no need for additional dependencies, you just need GeoIO to load the data